### ISSUE TITLE
After Successfully Running Code - Common Errors

### ISSUE THREAD

**Issue by Rana-0003** [steevenakintilo/TwitterGiveawayBot#3]
After Successfully Running Code for 1 hour, I'm getting these errors most:

DevTools listening on ws://........................
Hello world
Inside main one
Starting the program
Searching for Giveaway

Skipping unrecognised entry ID: 'promoted-tweet-166.............634-32..........0a8e'  ( More than 200 times )
Stopping after 20 empty pages ( 10 times )
Unavailable user in card on tweet 16716894.......153 ( 4-5 times )
Could not translate t.co card URL on tweet 16718465.......485 ( 3 times )
User 2789205119 not found in user refs in card on tweet 16712.........65762 ( 2 times )
Not a single successful participating giveaway yet.

These errors could be nothing but if this keeps on whole day than it could be something.

For new users, if you're catching up with my same issues, here's a little help:
Error "Stopping after 20 empty pages" Stopped/decreased 
( After correcting my mistake, as i'm searching giveaways in eng so in get_tweet.py i did lang:eng instead of lang:en - line 207-281 )

**Comment by steevenakintilo** [comment]
wait I've several question:
Is the bot on forever mode ? (check the configuration.yml for that)
How many giveaway have you been doing everytime you launched the bot?
are you on windows linux macos ?
those 4 error are commun so it's fine:
Stopping after 20 empty pages ( 10 times )
Unavailable user in card on tweet 16716894.......153 ( 4-5 times )
Could not translate t.co card URL on tweet 16718465.......485 ( 3 times )
User 2789205119 not found in user refs in card on tweet 16712.........65762 ( 2 times )

If I didn't make giveaway check if it did filled the recent_url.txt file if yes just go the configiration.yml and set the crash_or_no to true and It will make the giveaway from the recent_url.txt file

**Comment by Rana-0003** [comment]
I'm on windows linus macos and bot is on Forever Mode (True).
I've written almost 20 giveaways words (some of them are similars), never had any problem with that.
recent_url.txt is empty

**Comment by steevenakintilo** [comment]
Well if you have the 3 of them use linux it works better on linux idk why but it does
Oh If the recent_url.txt is empty that mean it didn't reach 1 giveaway
Try using linux and tell me if you still get those error

**Comment by Rana-0003** [comment]
Sorry to say but i'm getting same results on linux too.

Skipping unrecognised entry ID: 'promoted-tweet-167.............836-33..........9e5r' ( More than 200 times )
Not a single successful giveaway participation yet.

**Comment by steevenakintilo** [comment]
How is it possible I cloned the repo 20 minutes ago (testing on windows) and I got no error at all it.
Please share a screenshot so I can lookup on google and help you

**Comment by Rana-0003** [comment]
I tried it 2 hours ago. Let me try again

**Comment by steevenakintilo** [comment]
ok thanks

**Comment by Rana-0003** [comment]
Now without making any chances to your code (not even tried to edit fr lang into en) or unzipping cromedriver file into bot code folder. 
It's working now

DevTools listening on ws://...................
Hello world
Inside main one
Starting the program
Searching for Giveaway
Skipping unrecognised entry ID: 'promoted-tweet-1671521473587032073-3427651983bbe3cd'
Skipping unrecognised entry ID: 'promoted-tweet-1669412895170629634-34271801f63ecf60'
Skipping unrecognised entry ID: 'promoted-tweet-1628444039316897793-34276dcd2b19c36c'
Skipping unrecognised entry ID: 'promoted-tweet-1669412895170629634-34275545c4c85153'
Skipping unrecognised entry ID: 'promoted-tweet-1592723130035757057-342748b073eff89b'
Skipping unrecognised entry ID: 'promoted-tweet-1658088482244542465-3427c51fcfd970a2'
Skipping unrecognised entry ID: 'promoted-tweet-1669412895170629634-3427303e5e7599dd'
Skipping unrecognised entry ID: 'promoted-tweet-1666716857842245633-3427be0f4a03b8d4'
Skipping unrecognised entry ID: 'promoted-tweet-1628444039316897793-34270a211ef2dfd5'
Skipping unrecognised entry ID: 'promoted-tweet-1669412895170629634-34275749dd2d63dd'
Skipping unrecognised entry ID: 'promoted-tweet-1628444039316897793-342749390483f922'
Skipping unrecognised entry ID: 'promoted-tweet-1653785610820153344-3427459bcbfc6698'
Skipping unrecognised entry ID: 'promoted-tweet-1669412895170629634-3427cc5602990ecb'
Skipping unrecognised entry ID: 'promoted-tweet-1658088482244542465-342740a19db050e2'
Skipping unrecognised entry ID: 'promoted-tweet-1648979872281239552-342740d69efcc971'
Skipping unrecognised entry ID: 'promoted-tweet-1671521473587032073-34270d9115a69abf'
Skipping unrecognised entry ID: 'promoted-tweet-1669412895170629634-34274efcc4d4924f'
Skipping unrecognised entry ID: 'promoted-tweet-1628444039316897793-342756124a18c919'
Stopping after 20 empty pages
Stopping after 20 empty pages
Number of giveaway found = 3
Ending giveaway search
Connecting to ........ID........
Starting Twitter
button click
password done
login done
error
coockie done
error
notification done
error
coockie done
Giveaway number 0 / 3 all giveaway (even the one already done) 0


Found 3 giveways within 5 minutes in fr lang.

**Comment by steevenakintilo** [comment]
I know why have you modify the configuration.yml files if no you should read the readme to understand it better

**Comment by Rana-0003** [comment]
Yeah last query do i've to make changes in get_tweet.py line 207-281 like lang:fr to lang:en ?? There's nothing about it on readme.
And can we leave tagging/reply... Empty. As people can easily notice users using bot, bcoz bot use listed specific words or tagged people so through twitter search option people can easily find out, if a person using bot. Or In future can we hope to see old reply delete option.

Thanks for the support ❤️

**Comment by steevenakintilo** [comment]
Yes put it to english (en) I forgot about that I'm going to make little change and push it tonight.Like you want the bot to @ people withouth the sentence ???
I can make the two "features" you are asking it will take at max 5 minutes
What do you mean by that please explain "Or In future can we hope to see old reply delete option."

**Comment by steevenakintilo** [comment]
I think deleting old comment is not an hard task to do the only problem is that it would need to remove all the tweet or you can remove some of them but then you need to get the url of the tweet for that

**Comment by steevenakintilo** [comment]
I will try all of that when I finish work I had to do so it would be done tonight or tommorow not 100% sure

**Comment by steevenakintilo** [comment]
Added the language flux rss and option to add sentence on the configuration.yml file

**Comment by steevenakintilo** [comment]
Thanks to some of you idea the can now work on any language
